fx-src/FirefoxStorage.js

```diff
@@ -91,6 +97,7 @@ export default class FirefoxAdapter extends BaseAdapter {
   constructor(collection) {
     super();
     this.collection = collection;
+    this._connection = null;
```

Comment:
I think it doesn't harm, and makes it explicit we expect something to be stored here